docs/cypress.config.ts

```diff
@@ -8,23 +8,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       // implement node event listeners here
       on('task', {
-        readSitemapLinks: async () => {
-          if (fs.existsSync('./public/sitemap.xml')) {
-            const siteMapContent = fs.readFileSync(
-              './public/sitemap.xml',
-              'utf8'
-            );
-            const sitemapLinks: string[] = await sitemapUrls.extractUrls(
-              siteMapContent
-            );
-            return sitemapLinks.map((link) =>
-              link
-                .replace(`${BASE_URL}/`, '')
-                .replace('https://www.dev.ui.docs.amplify.aws/', '')
-                .replace('https://ui.docs.amplify.aws/', '')
-            );
-          }
-        },
```

Comment:
Cypress will record video for all tests by default, let's turn that off in config to save some memory, screenshots should be sufficient in case of test failure.

```
video: false
```

Reply:
Cypress also supports running tests in parallel, we should investigate if we can enable this in our CI. Ref: https://docs.cypress.io/guides/guides/parallelization

Reply:
Good points.
`video: false` added.

[parallelization](https://docs.cypress.io/guides/guides/parallelization) requires 1) multiple machines 2) multiple tests in separated files, so it doesn't help this PR 😢

Reply:
right, I should have been more clear, by `we should investigate if` I meant add a ticket for follow-up, not address in this PR.